src/pages/lessons/[slug].tsx

```diff
@@ -234,6 +248,7 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
             send('JOIN')
           }
         } else if (data.hls_url || data.dash_url) {
+          send('VIEW')
         } else {
```

Comment:
@joelhooks attempt to fix error caused by [this change](https://github.com/eggheadio/egghead-next/commit/5a12b76f336be3100d119d38f43c293c72a3f473#diff-167f71685e00411f721ebea7ded56fda1bf7a16e372f2c3c7e85a42e012c0b95R236-R237) in #256.